release-1.8/release_notes_draft.md

```diff
@@ -170,8 +170,8 @@ across the system. Here's the release [scalability validation report].
 
 * Advanced auditing has graduated from `v1alpha1` to `v1beta1` with the
   following changes to the default behavior.
-  * Advanced auditing is enabled by default.
-  * The `--audit-policy-file` is now required unless the `AdvancedAudit`
+  * Advanced auditing is the default auditing mechanism, which changes the auditing behavior
```

Comment:
`which changes the auditing behavior` why is this important? otherwise the sentence kind of lacks context

Reply:
Please focus also on the change a user experience after introducing advanced auditing.

Reply:
It's referencing https://github.com/kubernetes/kubernetes/issues/51899#issuecomment-331924016

We plan to add a section under "known issues" going into more detail on this.

Reply:
Actually, I wanted to highlight the fact that if you had audit logging previously enabled, the behavior will change in 1.8:
- The policy file is required (highlighted below)
- The output format is changed (also described below)

Reply:
@tallclair You just need to adjust the bullet points.

Advanced auditing is the default auditing mechanism. It introduces the following changes: